xrpl/models/ledger_objects/account_root.py

```diff
@@ -45,9 +149,14 @@ class AccountRoot(LedgerObject):
 class AccountRootFlags(Enum):
```

Comment:
This class does not appear to be used elsewhere in the codebase. Does it need to be included in the `type` information of the `flags` field?

Reply:
It's valid that a class isn't used elsewhere. I just added code to add ledger_objects flags to flag list.